### ISSUE TITLE
[FE] 10-10CG - Form Updates: Update 'Add a Caregiver' alert box

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#90692]
## Benefit to the Veteran
As a Veteran, I want to see clear messaging on alerts and errors, so that I can make decisions and adjustments to my input for a successful application.

## Background
Now that the July form updates have been released, we want to review content and make some adjustments to better align with VADS guidelines.
The change included in this ticket is listed below.  Refer to [Figma design file](https://www.figma.com/design/TxXD5bGUOhbHHWLb85GPjK/10-10CG?node-id=2424-43658&t=h6OMVME1QGjGuK0s-0) for visual guidance.

1. Update the alert box content

### Resources
- [Figma file](https://www.figma.com/design/TxXD5bGUOhbHHWLb85GPjK/10-10CG?node-id=2424-43658&t=h6OMVME1QGjGuK0s-0)
- [Mural board](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1683656893560/fccf821394d1cf693ad1c036e912e11a18ff88bd?sender=uadf1ed7fe7c76f0914967329)

---
## Tasks
- [x] Update the "You need to add a Family Caregiver" alert box
- [x] Review with the team/PM to confirm expected design/behavior
     - PM Signoff required to be documented in comments below
- [x] Inform [QA/Teammate] when the next step can begin

## Acceptance Criteria
- 10-10CG 'Add a Caregiver' alert box is updated

---
## How to configure this issue
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`1010-team`)
- [x] **Labeled with Form** (`1010-ez`, `1010-ezr`, `1010-cg`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `ux`, `design`, `research`, `product`, `qa`, `1010-analytics`)
- [x] **Labeled with Project** (`facilities-cg`, `registration`,  `toxic-exposure`, etc.)
- [x] **Add Assignees** (who will do this work?)
- [x] **Add Estimate effort  (or `needs refinement` label)
- [x] **Add Dependencies 'blocked by' and 'blocking'** (link tickets in the Dependencies section below)

**Comment by longmd** [comment]
Screenshot of the updated alert:

| Before | After |
| ------- | ------ |
| ![Screenshot 2024-09-03 at 13 15 05](https://github.com/user-attachments/assets/2dac2ffa-04f2-4dc9-ba69-aa08a6740726) | ![Screenshot 2024-09-03 at 13 13 40](https://github.com/user-attachments/assets/93888fed-f423-4f97-b2be-07668ce717a4) |

**Comment by hdjustice** [comment]
looking good!

**Comment by longmd** [comment]
@hdjustice / @allanto-ah , [here is the RI](http://cf17e6eda22c9bc2ebe9167282c77a7d.review.vetsgov-internal/family-and-caregiver-benefits/health-and-disability/comprehensive-assistance-for-family-caregivers/apply-form-10-10cg/) if you'd like to verify independently. Just let me know when I can ship it. 🚀

**Comment by allanto-ah** [comment]
Thank you, taking a look!

**Comment by longmd** [comment]
@allanto-ah , thats an interesting find. We will need to create a separate issue for that. Thats going to be a forms issue and outside of the scope of this work.

**Comment by allanto-ah** [comment]
Environment: RI

Verified that the content of the "Add a Caregiver" alert box has been updated and matches Figma. Looks good on my side! Once we get PM sign off, we should be good!

<img width="1023" alt="Screenshot 2024-09-04 at 7 24 33 AM" src="https://github.com/user-attachments/assets/107a9bee-3d12-41c2-b170-38dbbbeef758">

**Comment by allanto-ah** [comment]
Filed https://github.com/department-of-veterans-affairs/va.gov-team/issues/92075 for the above issue about the missing alert box on the "Review and submit" page

**Comment by hdjustice** [comment]
Looks good! I am good to move forward with deployment
